Allow for multiple twisted corners to be corrected (#64)
CarterKoala/wca-regulations-wrc: wca-regulations.md

    - 5b3) Permitted repairs:
        - 5b3a) If any parts have fallen out or moved out of place, the competitor may place them back.
        - 5b3b) If the competitor repairs the puzzle but finds that the puzzle is unsolvable later during the solve, they may disassemble and reassemble a maximum of 4 pieces to make the puzzle solvable.

>         - 5b3c) If the puzzle is unsolvable, and can be made solvable by rotating a single corner piece, the competitor may correct the corner piece by twisting it in place without disassembling the puzzle. If multiple corner pieces have been twisted as a result of puzzle defects, and the puzzle is not clearly solvable, the competitor may correct multiple corner pieces, so long as the number of pieces corrected by the competitor does not exceed the number of pieces twisted as a result of puzzle defects.

Reviewer: ```suggestion
        - 5b3c) If the puzzle is unsolvable, and can be made solvable by rotating a single corner piece, the competitor may correct the corner piece by twisting it in place without disassembling the puzzle. If multiple corner pieces have been twisted as a result of puzzle defects, the competitor may correct these twists individually, as long as the number of pieces corrected by the competitor does not exceed the number of pieces twisted as a result of puzzle defects.
```
IMO if a defect causes two corners to twist, then the competitor should be able to twist both of them back to their original position in every case. I don't think this allows anything that wouldn't be reasonable, as 5b2 still has to be followed

(Also, "as long" instead of "so long" to remain consistent with other instances in the regs)

Reviewer 2: I agree with Carter, together with 5b2 that should be sufficient. 

Lets say somebody got an OLL skip. They drop the cube and get two corner twists that result in an easily recognizable OLL. Would fixing these twists break 5b2? We could either say _yes_, because the cube was solvable, and performing the OLL would have been slower. We could also say _no_, because getting two twists and then fixing them again is still slower than not fixing the twists at all, so no advantage way gained. I'm in favor of the second solution, and therefore think Carter's wording of 5b3c is okay.

Reviewer: @ohexter thoughts?

Author: ```suggestion
        - 5b3c) If the puzzle is unsolvable, and can be made solvable by rotating a single corner piece, the competitor may correct the corner piece by twisting it in place without disassembling the puzzle. If multiple corner pieces have been twisted as a result of puzzle defects, the competitor may correct these twists, as long as the number of pieces corrected by the competitor does not exceed the number of pieces twisted as a result of puzzle defects.
```
OK with the logic change but can we remove "individually"? I don't see why a competitor shouldn't be able to fix both twists at the same time if they have the dexterity to do so, and if they're not allowed to do it we're effectively introducing a small artificial time penalty for having to do one and then the other.
